modules/core/shared/src/main/scala/lucuma/core/model/sequence/InstrumentConfig.scala

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.model.sequence
+
+import cats.Eq
+import cats.syntax.all._
+import lucuma.core.enum._
+import monocle.Focus
+import monocle.Lens
+import monocle.Prism
+import monocle.macros.GenPrism
+import org.typelevel.cats.time._
+
+import java.time.Duration
+
+sealed trait InstrumentConfig
+object InstrumentConfig {
+  final case class GmosNorth(
+    exposure: Duration,
```

Comment:
Are we in general leaning to `FiniteDuration` for exposure times? @swalker2m

Reply:
`FiniteDuration` extends `scala.concurrent.duration.Duration` class which states that `This class is not meant as a general purpose representation of time, it is optimized for the needs of scala.concurrent.`.

So `java.time.Duration` seems approriate for modelling things.

Reply:
I think when this exact point came up before the answer was along the lines of "well, everyone else uses it this way".  I'm fine with switching to `Duration` though.